python/lsst/cp/verify/verifyBfk.py

```diff
@@ -0,0 +1,237 @@
+        outputStatistics['SIZE_DIFF'] = sizeDiff
+
+        # This should have a debug view
+        return outputStatistics
+
+    def debugFit(self, stepname, xVector, yVector, yModel):
+        """Debug method for linearity fitting.
+        Parameters
+        ----------
+        stepname : `str`
+            A label to use to check if we care to debug at a given
+            line of code.
+        xVector : `numpy.array`, (N,)
+            The values to use as the independent variable in the
+            linearity fit.
+        yVector : `numpy.array`, (N,)
+            The values to use as the dependent variable in the
+            linearity fit.
+        yModel : `numpy.array`, (N,)
+            The values to use as the linearized result.
+        """
+        frame = getDebugFrame(self._display, stepname)
+        if frame:
+            import matplotlib.pyplot as plt
+            figure, axes = plt.subplots(1)
+
+            axes.scatter(xVector, yVector, c='blue', marker='+')
+            modelX = np.arange(np.min(xVector) - 1, np.max(xVector) + 1, 0.1)
+            axes.plot(modelX, exponentialModel(modelX, *yModel), 'r-')
+            plt.xlabel("Instrumental PSF magnitude")
+            plt.ylabel("Source size trace")
+            plt.title(f"BFK slope: {yModel[0]:.3f} + {yModel[1]:.3f} m")
+            figure.show()
+            prompt = "Press Enter or c to continue [chp]..."
+
+            while True:
+                ans = input(prompt).lower()
+                if ans in ("", " ", "c",):
+                    break
+                elif ans in ("p", ):
+                    import pdb
+                    pdb.set_trace()
+                elif ans in ("h", ):
+                    print("[h]elp [c]ontinue [p]db e[x]itDebug")
+            plt.close()
+
+    def verify(self, exposure, statisticsDict):
+        """Verify that the measured statistics meet the verification criteria.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            The exposure the statistics are from.
+        statisticsDictionary : `dict` [`str`, `dict` [`str`, scalar]],
+            Dictionary of measured statistics.  The inner dictionary
+            should have keys that are statistic names (`str`) with
+            values that are some sort of scalar (`int` or `float` are
+            the mostly likely types).
+
+        Returns
+        -------
+        outputStatistics : `dict` [`str`, `dict` [`str`, `bool`]]
+            A dictionary indexed by the amplifier name, containing
+            dictionaries of the verification criteria.
+        success : `bool`
+            A boolean indicating if all tests have passed.
+        """
+        verifyStats = {}
+        success = True
+        catalogVerify = {}
+        catStats = statisticsDict['CATALOG']
+
+        catalogVerify['BRIGHT_SLOPE'] = False
+        catalogVerify['NUM_MATCHES'] = False
+        # These values need justification.
```

Comment:
People should be rewarded for leaving comments like this, rather than punished, but reading it I still can't really help but ask... so, what's the justification for these values? (sorry!)

Reply:
`NUM_MATCHES > 10` exists because we need to be able to trust the fit.  In any real science image, this should be trivially met (the majority of my tests had ~200-300).
`BRIGHT_SLOPE < -0.5` is entirely arbitrary.  `alpha` must be negative (smaller size difference at larger magnitudes/fainter fluxes), but making it be simply negative ignores that the size change should be significant.  If `alpha =~ -0.01` we're essentially doing no correction.  Every other constraint I can think if is circular, because it requires trying to figure out how much smaller the kernel should make things.
I am open to any suggestion on this.

Reply:
Yeah, I don't really have any great suggestions, just had to comment on the comment really. Maybe just pasting this into a comment and replacing the `# These values need justification.` line would do it. And that way, if there's verifications failing in future, we can see that perhaps these values should be adjusted, as opposed to tests failing. (The other potential problem I can imagine is that if there's not much BF present in the first place (due to low depth in the observations, for example), or different sensors, (because we're camera-agnostic, haha), then this could cause a fail here, so noting that these values shouldn't be taken as gospel would be good).